stable/kibana/templates/service.yaml

```diff
@@ -6,6 +6,9 @@ metadata:
     chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
+    {{- range $key, $value := .Values.service.labels }}
```

Comment:
in keeping with helm defaults, can you outdent this line and the `{{- end }} line two spaces please? in general this makes the chart more readable.